mcs/resource_mcs_db_database.go

```diff
@@ -164,6 +163,18 @@ func resourceDatabaseDatabaseRead(d *schema.ResourceData, meta interface{}) erro
 		dbmsType = dbmsTypeInstance
 	}
 
+	if dbmsType == dbmsTypeInstance {
+		_, err := instanceGet(DatabaseV1Client, dbmsID).extract()
+		if err != nil {
+			return checkDeleted(d, err, "Error retrieving mcs_db_database")
+		}
+	} else {
+		_, err := dbClusterGet(DatabaseV1Client, dbmsID).extract()
+		if err != nil {
+			return checkDeleted(d, err, "Error retrieving mcs_db_database")
```

Comment:
you can use one `if err != nil` for both cases after if-else block, let's move it outside please

Reply:
done